paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/BacsMandateConfirmationFormScreenshotTest.kt

```diff
@@ -19,32 +18,70 @@ class BacsMandateConfirmationFormScreenshotTest {
     )
 
     @Test
-    fun testState() {
+    fun testFullForm() {
```

Comment:
I vote we just have one variant of paparazziRule for this.

Maybe something like 
```
    @get:Rule
    val paparazziRule = PaparazziRule(
        arrayOf(SystemAppearance.LightTheme),
        arrayOf(PaymentSheetAppearance.DefaultAppearance),
        arrayOf(FontSize.LargeFont),
    )
```

Reply:
We already have all the individual bits tested independently.